lib/dolla/encryption_tools.rb

```diff
@@ -0,0 +1,30 @@
+require 'hmac-sha1'
+require 'crypt/rijndael'
+
+module Dolla
+  module EncryptionTools
+    def build_digest *ary
+      sha1  = HMAC::SHA1.new Dolla.configuration.hmac_key
```

Comment:
We need to update this to use OpenSSL but it can be done on a different task/pr

Reply:
we will need to do a full test to see if it doesnt mess up the expected key by the SOAP service